SoftLayer/CLI/autoscale/create.py

```diff
@@ -0,0 +1,137 @@
+@click.option('--policy-name', help="Collection of policies for this group. This can be empty.")
+@click.option('--policy-amount', help="The number to scale by. This number has different meanings based on type.")
+@click.option('--userdata', help="User defined metadata string")
+@helpers.multi_option('--key', '-k', help="SSH keys to add to the root user")
+@helpers.multi_option('--disk', help="Disk sizes")
+@environment.pass_env
+def cli(env, **args):
+    """Order/create autoscale."""
+    scale = AutoScaleManager(env.client)
+    network = SoftLayer.NetworkManager(env.client)
+
+    pods = network.get_closed_pods()
+    closure = []
+
+    datacenter = network.get_datacenter(args.get('datacenter'))
+
+    ssh_keys = []
+    for key in args.get('key'):
+        resolver = SoftLayer.SshKeyManager(env.client).resolve_ids
+        key_id = helpers.resolve_id(resolver, key, 'SshKey')
+        ssh_keys.append(key_id)
+    scale_actions = [
+        {
+            "amount": args['policy_amount'],
+            "scaleType": args['policy_relative']
+        }
+    ]
+    policy_template = {
+        'name': args['policy_name'],
+        'policies': scale_actions
+
+    }
+    policies = []
+
+    block = []
+    number_disk = 0
+    for guest_disk in args['disk']:
+        disks = {'diskImage': {'capacity': guest_disk}, 'device': number_disk}
+        block.append(disks)
+        number_disk += 1
+
+    virt_template = {
+        'localDiskFlag': False,
+        'domain': args['domain'],
+        'hostname': args['hostname'],
+        'sshKeys': ssh_keys,
+        'postInstallScriptUri': args.get('postinstall'),
+        'operatingSystemReferenceCode': args['os'],
+        'maxMemory': args.get('memory'),
+        'datacenter': {'id': datacenter[0]['id']},
+        'startCpus': args.get('cpu'),
+        'blockDevices': block,
+        'hourlyBillingFlag': True,
+        'privateNetworkOnlyFlag': False,
+        'networkComponents': [{'maxSpeed': 100}],
+        'typeId': 1,
+        'userData': [{
+            'value': args.get('userdata')
+        }],
+        'networkVlans': [],
+
+    }
+
+    order = {
+        'name': args['name'],
+        'cooldown': args['cooldown'],
+        'maximumMemberCount': args['maximum'],
+        'minimumMemberCount': args['minimum'],
+        'regionalGroupId': args['regional'],
+        'suspendedFlag': False,
+        'balancedTerminationFlag': False,
+        'virtualGuestMemberTemplate': virt_template,
+        'virtualGuestMemberCount': 0,
+        'policies': policies.append(clean_dict(policy_template)),
+        'terminationPolicyId': args['termination_policy']
+    }
+
+    # print(virt_template)
+
+    for pod in pods:
+        if args.get('datacenter') in str(pod['name']):
+            closure.append(pod['name'])
+    click.secho(click.style('Warning: Closed soon: %s' % (', '.join(closure)), fg='yellow'))
```

Comment:
Lets only output this  datacenter closure warning if the user is creating a scale group in one of the pods to be closing. Might be confusing otherwise.